docs/dev-env-first-time-contributors.md

```diff
@@ -294,7 +294,24 @@ Welcome to Ubuntu 14.04.1 LTS (GNU/Linux 4.4.0-21-generic x86_64)
 Congrats, you're now inside the Zulip dev environment!
 
 You can confirm this by looking at the command prompt, which starts with
-`(zulip-venv)`.
+`(zulip-venv)`. In case it does not, your development environment might have halted
+while provisioning the vagrant guest. Check the `/var/log/zulip_provision.log`
```

Comment:
This is good content, but I'm not excited about having a long discussion of troubleshooting details inline in the middle of the flow, since it's a distraction for folks who don't run into this issue.  @christi3k do you have ideas for what the best way to handle this is?  I'm thinking maybe of moving all but the last sentence to be in the troubleshooting section of the doc and just link to that from here...

￼